test/runtime/stores/formatters.test.ts

```diff
@@ -39,8 +40,22 @@ addMessages('pt', require('../../fixtures/pt.json'));
 addMessages('pt-BR', require('../../fixtures/pt-BR.json'));
 addMessages('pt-PT', require('../../fixtures/pt-PT.json'));
 
+let onMissingMessageHandlerCalls: Array<{
+  lngs: string[];
```

Comment:
No need for this type of abbreviation 🙏 

```suggestion
  langs: string[];
```